hooks/eslint/index.js

```diff
@@ -32,8 +31,11 @@ const addEslintFileSuccess = (cwd, folderName) => {
     stdio:'inherit'
   }
 
-  return spawn(command, [], terminalOpts)
+  return spawnPromise(command, [], terminalOpts)
     .then(() => installPackagesSuccess(cwd, folderName))
+    .catch((error) => {
```

Comment:
I think add error handling is not related with this particular PR, we could open an issue for that